src/main/java/frc/robot/scoreassist/ScoreAssist.java

```diff
@@ -197,19 +198,19 @@ public boolean isAtPathTargetPose() {
    */
   @AutoLogOutput(key = "ScoreAssist/shouldOverridePath")
   public boolean shouldOverridePath() {
-    double translationStickMag =
-        Math.abs(
-            Math.hypot(
-                RobotContainer.driverControls.getLeftX(),
-                RobotContainer.driverControls.getLeftY()));
-
-    return translationStickMag > 0.15;
+    return !RobotContainer.autoScorePathing
+        || Math.abs(
+                Math.hypot(
+                    RobotContainer.driverControls.getLeftX(),
+                    RobotContainer.driverControls.getLeftY()))
+            > 0.15;
```

Comment:
Idk if this will actually help in a tangible way, but changing the order of these booleans means less operations are likely to happens during runtime